api/v1/version.go

```diff
@@ -79,7 +79,7 @@ const (
 	// Starting in v24.2.0, vcluster scrutinize command can read the
 	// database password from secret(k8s, aws, gsm)
 	ScrutinizeDBPasswdInSecretMinVersion = "v24.2.0"
-	// Starting in v24.3.0, sandboxing a subcluster is supported
+	// Starting in v24.3.0, sandboxing a subcluster  is supported
```

Comment:
I think your earlier version of this comment was better.
```suggestion
	// Starting in v24.3.0, sandboxing a subcluster with the operator is supported
```